saleor/payment/gateways/adyen/webhooks.py

```diff
@@ -292,7 +294,9 @@ def handle_authorization(notification: dict[str, Any], gateway_config: GatewayCo
             amount.get("value"), amount.get("currency")
         )
     except TypeError as e:
-        logger.exception("Cannot convert amount from minor unit", extra={"error": e})
+        logger.exception(
+            "Cannot convert amount from minor unit", extra={"error": str(e)}
+        )
```

Comment:
This looks wrong, `logger.exception()` automatically captures exception info, it should not need any extra copies.

Reply:
Resolved at e889bd62c289f8511ceb2492648dfabc0b0f9d81